scripts/install_docker.sh

```diff
@@ -3,3 +3,5 @@ set -ex
 
 curl -sSL https://get.docker.com/ubuntu/ | sh
 sudo gpasswd -a vagrant docker
+docker build -t android-base /vagrant/docker/android-base
+docker build -t android-studio /vagrant/docker/android-studio
```

Comment:
Then we should also update the shell scripts `build.sh` and `run.sh` because they refer to those name as well (do we actually need those? they don't seemed to be called from anywhere).